package.json

```diff
@@ -1,23 +1,23 @@
 {
-    "name": "ArmCord",
+    "name": "armcord",
```

Comment:
Changing the package name here also changes the path to config directory (from `~/.config/ArmCord` to `~/.config/armcord`), so users will have a dangling old config directory and will have to re-login after the update. Not a biggie, just thought should raise it up here in case if it's seen as a problem to someone.

Reply:
We saw that but weren't actually sure if it would be an issue. On windows it shouldn't be because paths are case-insensitive. But yeah on Linux this could be a minor issue

Reply:
I suggest we add code that checks if there's "ArmCord" already present and make the app use that.

Reply:
0329b2e